### PR TITLE
XS1 component

### PR DESCRIPTION
**Pull request by markusressel**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19115

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/xs1.markdown [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+title: "EZcontrol XS1"
+description: "Instructions on how to integrate an XS1 Gateway within Home Assistant."
+date: 2018-12-04 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xs1.jpg
+ha_category: Hub
+featured: true
```
Comment (MartinHjelmare) [inline source/_components/xs1.markdown]: Set to false.

File: source/_components/xs1.markdown [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ha_iot_class: "Local Polling"
+---
```
Comment (MartinHjelmare) [inline source/_components/xs1.markdown:19]: Please add `ha_release`.

Comment (MartinHjelmare) [reply source/_components/xs1.markdown:19]: If there's not much info in the platform pages, we now prefer to not create those pages but just create artifical entries in the component index by adding categories and redirects here in the component page front matter.

See eg:
https://github.com/home-assistant/home-assistant.io/pull/8427

File: source/_components/xs1.markdown [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+```yaml
+# Example configuration.yaml entry
+xs1:
+  host: "192.168.2.100"
+```
+
+The component will automatically detect the configuration of the XS1 Gateway only **on initialization** which currently means when Home Assistant boots. When you change the configuration of the XS1 you (currently) have to restart Home Assistant to see the effects.
+
+{% configuration %}
+xs1:
```
Comment (MartinHjelmare) [inline source/_components/xs1.markdown]: We don't include this key.

File: source/_components/xs1.markdown (hunk 1 above)
Comment (markusressel) [reply source/_components/xs1.markdown]: done

File: source/_components/xs1.markdown (hunk 3 above)
Comment (markusressel) [reply source/_components/xs1.markdown]: removed

File: source/_components/xs1.markdown (hunk 2 above)
Comment (markusressel) [reply source/_components/xs1.markdown:19]: ok

File: source/_components/xs1.markdown [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+sharing: true
+footer: true
+logo: xs1.jpg
+ha_category: 
+  - Hub
+  - Climate
+  - Sensor
+  - Switch
+redirect_from:
+  - /components/climate.xs1/
```
Comment (MartinHjelmare) [inline source/_components/xs1.markdown]: I guess we only need the redirects if there were already pages. I think we can remove them. Hopefully the index will still populate.

Comment (markusressel) [reply source/_components/xs1.markdown]: Seems like it doesn't? I don't understand what the build error message is trying to tell me :disappointed:

Comment (MartinHjelmare) [reply source/_components/xs1.markdown]: My guess was that it failed due to the redirects looking for pages that didn't exist. But I guess I'm wrong.

@klaasnicolaas any idea?

Comment (klaasnicolaas) [reply source/_components/xs1.markdown]: `redirect_from` is only for pages that already exist.

Comment (markusressel) [reply source/_components/xs1.markdown]: @klaasnicolaas I removed that already but it still doesn't compile :cry:

**Comment by klaasnicolaas** [comment]
Pherhaps that @balloob can take a look at the deploy error

**Comment by frenck** [comment]
Rebased, awaiting build results.

**Comment by frenck** [comment]
Thanks for the PR @markusressel! 🎉 🥇

**Review by frenck — APPROVED** [review]

**Pull request merged**